packages/next-plugin-google-analytics/src/document-head-tags.js

```diff
@@ -0,0 +1,22 @@
+export default async function headTags () {
+  return (
+    <>
+      <script
+        async
+        src={`https://www.googletagmanager.com/gtag/js?id=${
+          process.env.GA_TRACKING_ID
+        }`}
+      />
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+      window.dataLayer = window.dataLayer || [];
```

Comment:
Was the indentation messed up by prettier?

Reply:
Probably.